protomap/proto_test.go

```diff
@@ -289,3 +289,53 @@ func TestPathsFromProtoInternal(t *testing.T) {
 		})
 	}
 }
+
+func TestProtoFromPathsInternal(t *testing.T) {
+	tests := []struct {
+		desc             string
+		inProto          proto.Message
+		inVals           map[*gpb.Path]interface{}
+		inBasePath       *gpb.Path
+		wantProto        proto.Message
+		wantErrSubstring string
+	}{{
+		desc:    "string field",
+		inProto: &epb.ExampleMessage{},
+		inVals: map[*gpb.Path]interface{}{
+			mustPath("/string"): "hello",
+		},
```

Comment:
Add other longer/shorter paths to test the filtering?

Reply:
Done, thanks.